panther_analysis_tool/main.py

```diff
@@ -316,6 +191,47 @@ def _convert_keys_to_lowercase(mapping: Dict[str, Any]) -> Dict[str, Any]:
     return {k.lower(): v for k, v in mapping.items()}
 
 
+def zip_analysis_chunks(args: argparse.Namespace) -> List[str]:
+    logging.info("Zipping analysis items in %s to %s", args.path, args.out)
+
+    current_time = datetime.now().isoformat(timespec="seconds").replace(":", "-")
+    zip_chunks = [
+        # note: all the files we care about have an AnalysisType field in their yml
+        # so we can ignore file patterns and leave them empty
+        ZipChunk(patterns=[], types=(DATAMODEL, RULE, POLICY, PACK, GLOBAL)),  # type: ignore
+        ZipChunk(patterns=[], types=(QUERY, SCHEDULED_RULE)),  # type: ignore
+        ZipChunk(patterns=[], types=LOOKUP_TABLE),  # type: ignore
+    ]
+
+    filenames = []
+    chunks = analysis_chunks(ZipArgs.from_args(args), zip_chunks)
+    if len(chunks) != len(zip_chunks):
+        logging.error("something went wrong")
+        return []
+    for idx, chunk in enumerate(chunks):
+        filename = f"panther-analysis-{current_time}-chunk-{idx}.zip".format()
+        filename = add_path_to_filename(args.out, filename)
+        filenames.append(filename)
+        with zipfile.ZipFile(filename, "w", zipfile.ZIP_DEFLATED) as zip_out:
+            for name in chunk.files:
+                zip_out.write(name)
+
+    return filenames
+
+
+def add_path_to_filename(output_path: str, filename: str) -> str:
+    if output_path:
+        if not os.path.isdir(output_path):
+            logging.info(
+                "Creating directory: %s",
+                output_path,
+            )
+            os.makedirs(output_path)
+        filename = output_path.rstrip("/") + "/" + filename
```

Comment:
```suggestion
        filename = f"{output_path.rstrip('/')}/{filename}"
```
This might be neater?

Reply:
oh this was just a copy pasta .. yeah I can update